setup.py

```diff
@@ -74,42 +74,7 @@
     "Flask-BabelEx>=0.9.3",
     "Flask-Debugtoolbar>=0.10.1",
     "Flask-Login==0.4.1",
-    # --- `invenio` to fix broken deps -------------------------------------
-    #"invenio[base,auth]==3.2.0a9",
-    # NOTE: put back invenio[base,auth]>=3.3.0 when released and remove this
-    'Flask>=1.0.4',
-    'invenio-app>=1.2.3,<1.3.0',
-    'invenio-base>=1.2.0,<1.3.0',
-    'invenio-cache>=1.0.0,<1.1.0',
-    'invenio-celery>=1.1.1,<1.3.0',
-    'invenio-config>=1.0.2,<1.1.0',
-    "invenio-i18n>=1.1.1,<1.3.0",
-    # --- `base` bundle to fix broken deps ---------------------------------
-    # NOTE: put back invenio[base,auth]>=3.3.0 when released and remove this
-    'invenio-admin>=1.1.1,<1.3.0',
-    'invenio-assets>=1.1.3,<1.2.0',
-    'invenio-formatter>=1.0.2,<1.1.0',
-    'invenio-logging>=1.1.0,<1.3.0',
-    'invenio-mail>=1.0.2,<1.1.0',
-    'invenio-rest>=1.1.0,<1.3.0',
-    'invenio-theme>=1.1.4,<1.2.0',
-    # --- `auth` bundle to fix broken deps ---------------------------------
-    # NOTE: put back invenio[base,auth]>=3.3.0 when released
-    'invenio-access>=1.3.0,<1.4.0',
-    # NOTE: put me back when topical branches removed
-    #'invenio-accounts>=1.1.1,<1.3.0',
-    'invenio-oauth2server>=1.0.3,<1.3.0',
-    # NOTE: put me back when topical branches removed
-    #'invenio-oauthclient>=1.1.3,<1.3.0',
-    'invenio-userprofiles>=1.0.1,<1.2.0',
-    # --- `metadata` bundle without records UI -----------------------------
-    "invenio-indexer>=1.1.0,<1.2.0",
-    "invenio-jsonschemas>=1.0.0,<1.1.0",
-    "invenio-pidstore>=1.2.0,<1.3.0",
-    "invenio-records-rest>=1.7.0,<1.8.0",
-    "invenio-records>=1.3.0,<1.4.0",
-    # --- `files` bundle with only invenio-files-rest ----------------------
-    "invenio-files-rest>=1.0.6,<1.1.0",
+    "invenio[base,auth,metadata,files]>=3.3.0,<3.4.0",
```

Comment:
there was invenio-accounts changes released in this, did you check if we can still login and POST forms (I believe also csrf token was added - not sure) after this change?

Reply:
we cannot use the full `metadata` bundle, you need to "explode" it and remove the UI modules. The same might be for `files`, to be checked.
See comments that I have left:
`# NOTE: put back invenio[base,auth]>=3.3.0 when released and remove this`
vs
`# --- `metadata` bundle without records UI -----------------------------`
`# --- `files` bundle with only invenio-files-rest ----------------------`